src/ui/load.js

```diff
@@ -26,44 +30,45 @@ export async function load(): Promise<LoadResult> {
   // than ternaries. There's also a lot of repeated code here
 
   const diskStorage = createLedgerDiskStorage("data/ledger.json");
+  const networkStorage = new NetworkStorage("");
   const ledgerManager = new LedgerManager({
     storage: diskStorage,
   });
 
   const queries = [
-    fetch("sourcecred.json"),
-    fetch("static/server-info.json"),
-    fetch("config/currencyDetails.json"),
-    fetch("output/credGraph.json"),
+    loadJson(networkStorage, "sourcecred.json", (Combo.raw: any)),
```

Comment:
https://github.com/sourcecred/sourcecred/blob/master/src/api/instanceConfig.js#L37

Reply:
I had to split out the raw parser from the bundledplugins parser, since the bundledplugins parser actually reaches into node fs modules.